src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  background-color: #000;
+}
+
+.external,
+.middle,
+.interior {
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.external {
+  width: 350px;
+  background-color: #308dc0;
+  box-sizing: border-box;
```

Comment:
this property is usually assigned with `*` selector so that every element on the page have it